go.mod

```diff
@@ -16,23 +17,29 @@ require (
 	github.com/zclconf/go-cty v1.11.1
 	golang.org/x/exp v0.0.0-20220907003533-145caa8ea1d0
 	google.golang.org/genproto v0.0.0-20221014213838-99cd37c6964a
-	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f
+	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c
```

Comment:
Any idea why this rolled? Mostly just curious. It is updated to the latest version, which is still from 2020 so I see no issue here.